packages/types/api-extractor.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../api-extractor.packages.json",
+  "mainEntryPointFilePath": "./dist-types/index.d.ts"
+}
```

Comment:
nit: The formatting CI check is failing on the lack of newline at the end of this.